doc/_templates/index.html

```diff
@@ -74,9 +74,9 @@ <h2 style="margin-bottom: 0">{%trans%}Documentation{%endtrans%}</h2>
 
   <p>{%trans%}
     You can also download PDF/EPUB versions of the Sphinx documentation:
-    a <a href="https://media.readthedocs.org/pdf/sphinx/stable/sphinx.pdf">PDF version</a> generated from
+    a <a href="https://media.readthedocs.org/pdf/sphinx/master/sphinx.pdf">PDF version</a> generated from
     the LaTeX Sphinx producer, and
-    an <a href="https://media.readthedocs.org/epub/sphinx/stable/sphinx.epub">EPUB version</a>.
+    an <a href="https://media.readthedocs.org/epub/sphinx/master/sphinx.epub">EPUB version</a>.
```

Comment:
@shimizukawa do you have any idea to generate appropriate URL? It would be much better if we can use branch name to here.

Reply:
These are the URL patterns for PDF that generated on RTD.

- http://readthedocs.org/projects/sphinx/downloads/pdf/master/
- http://readthedocs.org/projects/sphinx/downloads/pdf/1.7/
- http://readthedocs.org/projects/sphinx-ja/downloads/pdf/1.7/

Pattern

- `http://readthedocs.org/projects/<PROJECT-NAME-ON-RTD>/downloads/pdf/<VERSION-NAME-ON-RTD>/`

Honestly, it is difficult to generate this URL pattern on building time from sphinx. 

The links for PDF/EPUB was necessary before relocation to RTD, but I think that it is unnecessary now because document readers can download PDF from RTD control panel.

![image](https://user-images.githubusercontent.com/151623/41505201-730014ba-723e-11e8-9b8e-89becdfdab42.png)

So I suggest that it is time to remove these links from Sphinx documentation.

Reply:
Absolutely. Let's remove them from contents part!